proto-public/pbauth/v2beta1/traffic_permissions.proto

```diff
@@ -114,7 +114,7 @@ message ExcludePermissionRule {
   // Methods is the list of HTTP methods.
   repeated string methods = 4;
 
-  DestinationRuleHeader header = 5;
+  repeated DestinationRuleHeader headers = 5;
```

Comment:
It looks like `exclude` doesn't exist on the `DestinationRuleHeader` message below.

Reply:
Yes, I de-scoped ExcludePermissionRules for this change, because the rule comparisons get to be quite tricky (regex filter combinations for example) - because you can expand to two sets and then nullify them in the same way as identities. The exclude rules are going to require a lot of testing and careful work.